cinema/views.py

```diff
@@ -56,3 +128,32 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def list(self, request, *args, **kwargs):
+        response = super().list(request, *args, **kwargs)
+        return Response(response.data["results"])
```

Comment:
The `list` method overrides the default behavior and removes pagination metadata by returning only `response.data['results']`. Consider returning the full `response.data` to include pagination metadata.